webui/src/pages/user/user-namespace-extension-list.tsx

```diff
@@ -0,0 +1,58 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+import * as React from 'react';
+import { Theme, createStyles, WithStyles, withStyles, Typography } from '@material-ui/core';
+import { Namespace } from '../../extension-registry-types';
+import { UserNamespaceExtensionListItem } from './user-namespace-extension-list-item';
+import { ExtensionRegistryService } from '../../extension-registry-service';
+import { ErrorResponse } from '../../server-request';
+
+const extensionListStyles = (theme: Theme) => createStyles({
+    container: {
+        padding: theme.spacing(5, 0),
+    },
+    extensions: {
+        display: 'grid',
+        gridTemplateColumns: `repeat(auto-fit, minmax(200px, 1fr))`,
+        gap: `1.5rem`,
+        marginTop: '1rem',
+    }
+});
+
+const UserNamespaceExtensionListComponent = ({ namespaces, service, setError, classes }: UserNamespaceExtensionListComponent.Props) => {
+    const extensions = Object.keys(namespaces[0].extensions).map((key: any) => namespaces[0].extensions[key]);
```

Comment:
`console.log(typeof Object.keys(namespaces[0].extensions)[0], Object.keys(namespaces[0].extensions)[0])` this outputs `string cpptools` in the console but when I try as the type `string` instead of `any` I get the following error:
```ts
Element implicitly has an 'any' type because index expression is not of type 'number'.ts(7015)
```

Reply:
There is a false type on the `Namespace` interface. 
The type of the `Extensions` field should be `{ [key: string]: string }` instead of `Extension[]`. Typescript thinks it is an array and so it expects a number. 
Could you please change this? Thanks.